internal/notify/client.go

```diff
@@ -208,3 +246,12 @@ func (c *Client) do(req *http.Request) (response, error) {
 
 	return r, nil
 }
+
+func (c *Client) makeReference(to string, email Email) string {
```

Comment:
I don't think there is any email we'd want to send to the same person multiple times, right?